Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewController.swift

```diff
@@ -110,7 +111,18 @@ public final class GDSInformationViewController: BaseViewController, TitledViewC
             }
         }
     }
-    
+
+    /// Stack View: `UIStackView`. Any `UIView` which is on the `GDSInformationViewModelWithChildView` view model's `childView` property.
+    /// This will be added to the `stackView` below the existing `bodyLabel`
+    @IBOutlet private var stackView: UIStackView! {
+        didSet {
+            if let childView = (viewModel as? GDSInformationViewModelWithChildView)?.childView {
+                stackView.addArrangedSubview(childView)
+                stackView.accessibilityIdentifier = "information-optional-stack-view"
+            }
+        }
+    }
```

Comment:
Think this should probably be moved up the file to fit the rest of the class (mirroring the flow of the view)